frameworks/hdfs/tests/conftest.py

```diff
@@ -3,10 +3,14 @@
 import sdk_security
 from tests import config
 
+from sdk_utils import random_string
+
 
 @pytest.fixture(scope='session')
-def configure_universe():
-    yield from sdk_repository.universe_session()
+def configure_universe(tmpdir_factory):
```

Comment:
Since the `configure_universe` fixture is not dependent on any specific framework code, we could consider moving it into the root `conftest.py` as a fixture and then also adding the `autouse=True` parameter.